bin/git-ranch

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+echo "Howdy pardner! Looks like you've mistaken a git command. Did you mean \`git branch\`? See \`git --help\`. Yee-haw!
+
+While yer in the ranchin' mood, why don't ya try this recipe for bacon 'n beans?
+
+Ingredients
+
+1 pound 90% lean ground beef*
+1 pound bacon
+1 medium yellow onion, chopped
+2 (28 ounce) cans baked beans (I used Bush's Original)
+1 (15 ounce) can pork and beans
+2 (15 ounce) cans black beans, drained and rinsed
+1 cup barbecue sauce (I used Sweet Baby Rays)
+1/4 cup yellow mustard
+2 tablespoons Worcestershire sauce
+See a Quick Sausage Alfredo Recipe  
+ 
+Instructions
+
+Brown the ground beef in a large skillet. Remove to a large bowl and discard the fat.
+In the same skillet, cook the bacon. You can either chop it beforehand or crumble it afterward. Remove to a paper-towel lined plate. Discard all but 1 tablespoon of bacon grease from the pan.
+Preheat the oven to 400 degrees F.
```

Comment:
Thoughts on cooking the bacon *first*, leaving *all* the grease in the skillet, and cooking the beef in the bacon grease?